package.json

```diff
@@ -23,6 +23,7 @@
   "license": "ISC",
   "scripts": {
     "rebel:re": "npm run rebel && ./rebel.sh examples/reason-project",
+    "rebel:rs": "npm run rebel && ./rebel.sh examples/recursive-src",
```

Comment:
Great. Let's have actual integration tests later, as per the last section of README.